### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by Victoria-Moiseienko**

### CODE REVIEW

File: src/main/java/core/basesyntax/MainApp.java [hunk 1]
```diff
@@ -12,5 +12,17 @@
  * - Отправить все машини на работу и остановить работу всех машин.
  */
 public class MainApp {
+    public static void main(String[] args) {
+        Track track = new Track();
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/MainApp.java:16]: be attentive with [checklist](https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_02.html)
- Use abstract class to create Track, Bulldozer and Excavator instances.

Example:
Animal cat = new Cat();

Comment (Victoria-Moiseienko) [reply src/main/java/core/basesyntax/MainApp.java]: Done

File: src/main/java/core/basesyntax/MainApp.java [hunk 2]
```diff
@@ -13,16 +13,18 @@
...
 
-        track.doWork();
-        bulldozer.doWork();
-        excavator.doWork();
+        Machine[] arrayOfMachine = {track, bulldozer, excavator};
 
-        track.stopWork();
-        bulldozer.stopWork();
-        excavator.stopWork();
+        for (Machine amachine : arrayOfMachine) {
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/MainApp.java:22]:
```suggestion
        for (Machine machine : arrayOfMachine) {
```

File: src/main/java/core/basesyntax/MainApp.java [hunk 3]
```diff
@@ -13,16 +13,18 @@
...
+            amachine.doWork();
+        }
+
+        for (Machine amachine : arrayOfMachine) {
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/MainApp.java:26]:
```suggestion
        for (Machine machine : arrayOfMachine) {
```

File: src/main/java/core/basesyntax/MainApp.java (hunk 2 above)
Comment (Victoria-Moiseienko) [reply src/main/java/core/basesyntax/MainApp.java]: done

File: src/main/java/core/basesyntax/MainApp.java (hunk 3 above)
Comment (Victoria-Moiseienko) [reply src/main/java/core/basesyntax/MainApp.java]: done

**Pull request closed without merging**